backend/src/models/Tag.ts

```diff
@@ -19,6 +19,9 @@ export class TagModel {
   @prop({ required: false })
   public boardID!: string;
 
+  @prop({ required: false })
```

Comment:
This should be required and so should boardID which I'm not sure why it isn't already lol.